packages/backend/src/services/CsvService/CsvService.test.ts

```diff
@@ -99,7 +99,8 @@ describe('Csv service', () => {
         });
 
         expect(csvContent).toEqual(
-            `column number,column string,column date
+            // eslint-disable-next-line no-irregular-whitespace
+            `﻿column number,column string,column date
```

Comment:
Github doesn't show the irregular whitespace. 
This is because we are now injecting a BOM to the CSV file. https://github.com/lightdash/lightdash/pull/15269

<img width="751" alt="Screenshot 2025-06-11 at 18 18 53" src="https://github.com/user-attachments/assets/790917d5-d4b0-4b36-8ddf-e2836bff0042" />